.circleci/config.yml

```
@@ -343,6 +343,32 @@ jobs:
       - package-build:
           type: armhf
           nightly: << parameters.nightly >>
+  docker-nightly:
+    docker:
+      - image: docker:20.10.14-git
+    steps:
+      - run:
+          name: login to quay.io
+          command: echo "$QUAY_PASS" | docker login --username "$QUAY_USER" --password-stdin quay.io
```

```suggestion
          command: docker login --username "$QUAY_USER" --password "$QUAY_PASS" quay.io
```

I guess they both do the same thing, maybe this is more straightforward?

---

Yeah, I can swap it back for that. I had originally done that but a bunch of scary warnings show up telling you to pass it in the other way. However, since circle ci will hide the variable we should be good to go. with that.

---

Avoiding scary warnings is a good idea, if the CLI suggests using --password-stdin then that works for me.